containers/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM continuumio/miniconda
+COPY environment.yml /
+RUN conda config --add channels defaults \
```

Comment:
Can specify the channels in the `environment.yaml` file:

```yaml
channels:
  - conda-forge
  - bioconda
```

Reply:
Done, thanks Phil!